tests/conftest.py

```diff
@@ -33,16 +39,46 @@ def members():
     members = [
         [
             amureki_id,
-            "amureki"
+            'amureki'
         ],
         [
             sebastian_id,
-            "sebastiankapunkt"
+            'sebastiankapunkt'
         ],
         [
             anapaulagomes_id,
-            "anapaulagomes"
+            'anapaulagomes'
         ]
     ]
 
     return members
+
+
+@pytest.fixture
+def redis_clean_up():
+    redis_storage.delete(REDIS_KEY_SEND_FEEDBACK)
+
```

Comment:
Those empty lines look a little confusing. I think reading is easier if you remove them.